### PR TITLE
fix issues running the unit tests with bitcoin cash

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -4119,7 +4119,9 @@ bool ContextualCheckBlock(const CBlock &block, CValidationState &state, CBlockIn
 
     // BUIP055 enforce that the fork block is > 1MB
```
Comment (sickpig) [inline src/main.cpp:4120]: I would have kept the reference to BUIP055/UAHF specification.

File: src/test/script_tests.cpp [hunk 2]
```diff
@@ -787,9 +787,10 @@ BOOST_AUTO_TEST_CASE(script_PushData)
 }
 
 CScript
-sign_multisig(CScript scriptPubKey, std::vector<CKey> keys, CTransaction transaction)
+sign_multisig(const CScript& scriptPubKey, std::vector<CKey> keys, const CTransaction& transaction, CAmount amt)
```
Comment (sickpig) [inline src/test/script_tests.cpp:790]: I would have add an explanation of what we are trying to fix here. Form what I can understand after hard fork, replay protected tx's require the amount field to be non-zero for previous outputs in signed tx's.

Comment (sickpig) [reply src/test/script_tests.cpp:790]: I think we need fix also the qa test `rawtransactions.py` and the RPC `signrawtransaction`  to take into account of the mandatory presence of a non zero value for amount.

File: src/main.cpp (hunk 1 above)
Comment (gandrewstone) [reply src/main.cpp:4120]: you mean in the comment?  I kept this comment.

Comment (sickpig) [reply src/main.cpp:4120]: sorry to few caffeine, you are indeed right.
Maybe we should change BUIP055 into BUIP055/UAHF, cause we are actually following the UAHF spec.

File: src/test/script_tests.cpp (hunk 2 above)
Comment (gandrewstone) [reply src/test/script_tests.cpp:790]: yes you are right.  Fixed now

Comment (sickpig) [reply src/test/script_tests.cpp:790]: thanks

**Comment by sickpig** [comment]
still failing on `buip055.py` both for linux 64 and 32 bit.

https://travis-ci.org/BitcoinUnlimited/BitcoinUnlimited/jobs/262314125#L2551

**Review by ptschip — APPROVED** [review]

**Pull request merged**